MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,899 @@
+            {
+                e.DataFormItem.PlaceholderText = "Enter description";
+                e.DataFormItem.PlaceholderColor = Colors.MistyRose;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when AutoGenerate is true.
+
+For AutoGenerate false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change padding for auto-generated editor
+
+You can create a space around a label and editor by using `Padding` property in the DataFormItem by using `GenerateDataFormItem` event when `AutoGenerate` is true.
+
+Here, `FirstName` data field will be changed from the default position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.Padding = new Thickness(10);
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Change the label name for auto-generated editor
+
+By default property name will be displayed as label text. You can change the label text by using display attribute or using the `GenerateDataFormItem` event when `AutoGenerate` is true.
+
+#### Using attribute
+
+You can change the label text by setting the `Name` or `ShortName` in display attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    
+    [Display(Name = "FirstName")]
+    public string Name { get; set; }
+    
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+You can change the label text by using the `LabelText` property in `DataFormItem` .
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LabelText = "FirstName";
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+GenerateDataFormItem event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add reference link for explicit creaate data editor
+
+## Hide Label for an editor
+
+By default, the label will be generated by the property name. You can hide the label text by using the DataFormDisplayAttribute or using the `GenerateDataFormItem` event when `AutoGenerate` is true.
+
+For `AutoGenerate` false you can use property directly by using the DataFormItem - add refrence link for explict create data editor
```

Comment:
For `AutoGenerate` false you can use property directly by using the DataFormItem - add refrence link for explict create data editor

remove

Reply:
removed